### DISCUSSION TITLE
Permissions to disallow transfer?

### DISCUSSION THREAD

**Discussion by psybers** [community/community#64800] [Apps, API and Webhooks] [answered]
### Select Topic Area

Question

### Body

Organizations have a permission in "Member Privileges" to `Allow members to delete or transfer repositories for this organization`.

I have dug around a lot and tried searching everywhere.  Is there any way to set that permission to `false` via the REST API/CLI client?  Right now we have to set it manually via the web interface for any new org.

Most of the permissions on that page (`https://github.com/organizations/<org>/settings/member_privileges`, under "Member privileges") can be set, but I can't find the ones under "Admin repository permissions" and "Member team permissions" anywhere.

**Comment by arafajs** [comment, marked as answer]
Unfortunately, there is no way to set the `Allow members to delete or transfer repositories for this organization` permission to false via the REST API or CLI client. This permission is currently only available to be set via the web interface.
